third-party/cmake/BuildJeMalloc.cmake

```diff
@@ -16,7 +16,7 @@ ExternalProject_Add(jemalloc
     -DUSE_EXISTING_SRC_DIR=${USE_EXISTING_SRC_DIR}
     -P ${CMAKE_CURRENT_SOURCE_DIR}/cmake/DownloadAndExtractFile.cmake
   BUILD_IN_SOURCE 1
-  CONFIGURE_COMMAND ${DEPS_BUILD_DIR}/src/jemalloc/configure
+  CONFIGURE_COMMAND sh ${DEPS_BUILD_DIR}/src/jemalloc/autogen.sh && ${DEPS_BUILD_DIR}/src/jemalloc/configure
```

Comment:
what prompted this?

Reply:
When using the `.tar.bz2` from releases, there is no need for `sh autogen.sh`. I didn't had time to test it on my machine (I'm currently transitioning from Windows to Linux).

Reply:
Maybe I should remove the `${DEPS_BUILD_DIR}/src/jemalloc/configure` since `autogen.sh` will run `./configure`.

Reply:
Oh, yuck. Let's stick to releases only then, otherwise we'll get confused.

Releases tend to be post-processed for "end user" consumption. We don't want to have to micro-manage build steps if possible.